Sources/TuistGenerator/Linter/GraphLinter.swift

```diff
@@ -6,7 +6,7 @@ import TuistSupport
 
 public protocol GraphLinting: AnyObject {
     func lint(graphTraverser: GraphTraversing) -> [LintingIssue]
-    func lintCodeCoverageMode(_ mode: CodeCoverageMode?, graphTraverser: GraphTraversing) -> [LintingIssue]
+    func lintCodeCoverageMode(graphTraverser: GraphTraversing) -> [LintingIssue]
```

Comment:
minor: (can be a separate PR) - I believe now the options are within the `Workspace` type, there is no longer a need to have a operate lint method, this check can be done internally when calling `lint(graphTraverser: graphTraverser)`

Reply:
yep, I already planned to do that later